src/hyperfine/export/mod.rs

```diff
@@ -87,28 +92,29 @@ impl ExportManager {
     }
 
     /// Add an additional exporter to the ExportManager
-    pub fn add_exporter(&mut self, for_type: ExportType) {
-        self.exporters.push(for_type);
+    pub fn add_exporter(&mut self, export_type: ExportType, filename: &str) {
```

Comment:
After seeing this, your thoughts about splitting the filename from the enum makes more sense. And this looks really good. Is there a particular benefit we get from this?

Reply:
My thought was that we could save some duplication because we don't have to unpack the filename from each `ExportType`.